server/controllers/appointments/deleteAppointments.js

```diff
@@ -0,0 +1,18 @@
+const { deleteAppointmentsQueries } = require('../../database/queries');
+
+const deleteAppointments = async (req, res, next) => {
+  try {
+    const { appointmentId } = req.params;
+
+    const { rows } = await deleteAppointmentsQueries(appointmentId);
+    return res.json({
+      statusCode: 200,
+      message: 'appointment deleted successfully',
+      data: rows,
```

Comment:
I think it's not necessary to return the deleted patient to the client,It's good to make the res.body equal to :-1: 
```js
return res.json({
      statusCode: 200,
      message: 'appointment deleted successfully',})
```